### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onDidSave' of undefined

### ISSUE THREAD

**Issue by kandeshvari** [weskinner/symbol-gen#37]
[Enter steps to reproduce below:]

1. Open Settings Pane
2. Press ctrl+s

I understand that is abnormal to try to save settings but it's my finger-pattern :)

**Atom Version**: 1.0.2
**System**: Ubuntu 14.10
**Thrown From**: [symbol-gen](https://github.com/weskinner/symbol-gen) package, v1.0.0

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidSave' of undefined

```
At /home/dk/.atom/packages/symbol-gen/lib/symbol-gen-view.coffee:42

TypeError: Cannot read property 'onDidSave' of undefined
  at SymbolGenView.module.exports.SymbolGenView.check_for_on_save (/home/dk/.atom/packages/symbol-gen/lib/symbol-gen-view.coffee:42:43)
  at atom-workspace.<anonymous> (/home/dk/.atom/packages/symbol-gen/lib/symbol-gen-view.coffee:35:58)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -0:40.1.0 core:delete (atom-text-editor.editor.mini.is-focused)
  2x -0:13 core:save (input#find-and-replace.focusEditorAfterSearch.)
```

### Config

```json
{
  "core": {
    "themes": [
      "molecula-ui",
      "molecula-syntax"
    ],
    "disabledPackages": [
      "linter-python-pep8",
      "linter-python-pyflakes"
    ],
    "projectHome": "/home/dk/devel/atom"
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.8
atom-jinja2, v0.5.0
atom-runner, v2.4.1
autocomplete-plus-python-jedi, v0.3.6
file-icons, v1.5.8
git-diff-details, v0.18.0
git-log, v0.4.1
language-salt, v0.4.0
linter, v1.2.4
linter-coffeelint, v1.0.0
linter-flake8, v1.5.0
local-history, v3.2.3
minimap, v4.12.0
minimap-find-and-replace, v4.2.0
minimap-git-diff, v4.1.7
minimap-linter, v1.0.0
molecula-syntax, v0.0.1
molecula-ui, v1.0.1
open-recent, v2.3.0
project-manager, v1.15.11
symbol-gen, v1.0.0
terminal-panel, v1.14.1

# Dev
No dev packages
```

**Comment by Dbuggerx** [comment]
Same happening here.

**Comment by pchomik** [comment]
The same here and additionally the error appears when all tabs are closed.

Atom: 1.5.3
Package version: 1.0.0

**Comment by shemerey** [comment]
same for me, open atom without active editor cause this issue

**Comment by weskinner** [comment]
resolved in v1.3.0
